src/main/c/com_wynntils_antiope_core_DiscordGameSDKCore.h

```diff
@@ -39,14 +39,6 @@ JNIEXPORT void JNICALL Java_com_wynntils_antiope_core_DiscordGameSDKCore_destroy
 JNIEXPORT jlong JNICALL Java_com_wynntils_antiope_core_DiscordGameSDKCore_getActivityManager
   (JNIEnv *, jobject, jlong);
 
-/*
- * Class:     com_wynntils_antiope_core_DiscordGameSDKCore
- * Method:    getUserManager
- * Signature: (J)J
- */
-JNIEXPORT jlong JNICALL Java_com_wynntils_antiope_core_DiscordGameSDKCore_getUserManager
-  (JNIEnv *, jobject, jlong);
-
```

Comment:
Touching this directly is so so but you should remove
`private native long getUserManager(long pointer);` 
from the java file. Isn't there a C implementation that you should remove as well?